java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -95,4 +95,95 @@ public interface BitmapBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> setbit(String key, long offset, long value);
+
+    /**
+     * Return the position of the first bit matching the given <code>bit</code> value.
+     *
+     * @see <a href="https://redis.io/commands/bitpos/">redis.io</a> for details.
+     * @param key The key of the string.
+     * @param bit The bit value to match.
+     * @return The position of the first occurrence matching <code>bit</code> in the binary value of
+     *     the string held at <code>key</code>. If <code>bit</code> is not found, a <code>-1</code> is
+     *     returned.
+     * @example
+     *     <pre>{@code
+     * Long payload = client.bitpos("myKey1", 0).get();
+     * assert payload == 3L; // Indicates that the first occurrence of a 0 bit value of the string stored at "myKey1" is at the fourth position.
```

Comment:
Should we say "position 3" instead of "fourth position"? bitpos is short for bit position, and the returned position is 3. I know you are saying fourth position because it is 0 indexed, but "position 3" might be clearer given the context. Same applies below
```suggestion
     * assert payload == 3L; // Indicates that the first occurrence of a 0 bit value of the string stored at "myKey1" is at position 3.
```